src/applications/letters/containers/DownloadLetters.jsx

```diff
@@ -8,30 +8,27 @@ import SegmentedProgressBar from '@department-of-veterans-affairs/formation-reac
 import StepHeader from '../components/StepHeader';
 import { chapters } from '../routes';
 
-export class DownloadLetters extends React.Component {
-  render() {
-    const { children, location } = this.props;
-    const currentPageIndex = findIndex(['path', location.pathname], chapters);
-    const currentStep = currentPageIndex + 1;
+export function DownloadLetters({ children, location }) {
```

Comment:
@department-of-veterans-affairs/vsa-bam-1-frontend is `mapStateToProps` still needed here? It doesn't look like this component uses any of those props, but I don't have a thorough understanding of how the app works.

Reply:
I don't see how it would be used either...Unfortunately I don't think anyone has a ton of knowledge on this app though to say 100%. I wonder if it was just copied over from another component at some point and never removed.

Reply:
Also looks safe to remove to me. The last remaining mapped props were removed 10 months ago.

Reply:
I'm not sure either. Mr Sprinkle has worked with this app more than me.

Reply:
Definitely looks like they aren't being used. Safe to delete :)

Reply:
Once I'm finished with the linting rules I'll make another PR and link back to this.